src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthControllerTest.java

```diff
@@ -129,6 +133,85 @@ void test_health_reports_up_if_time_outside_of_working_hours_end_time() throws E
         verify(caseEventMessageRepository, never()).getNumberOfMessagesReceivedInLastHour(any());
     }
 
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        scripts = {"classpath:scripts/insert_case_event_messages_for_received_messages_check.sql"})
+    void test_health_reports_up_if_received_messages_in_last_hour() throws Exception {
+
+        // GIVEN
+        setClock(LocalDateTime.of(2024, 4, 2, 14,15));
+
+        // THEN
+        mockMvc.perform(get("/health"))
+            .andExpect(
+                jsonPath("$.components.ccdMessagesReceived.status")
+                    .value(UP.toString()))
+            .andExpect(jsonPath("$.components.ccdMessagesReceived.details." + CASE_EVENT_HANDLER_MESSAGE_HEALTH)
+                           .value(String.format(MESSAGES_RECEIVED, "test")));
+
+    }
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+        scripts = {"classpath:scripts/insert_case_event_messages_for_received_messages_check.sql"})
+    void test_health_reports_down_if_not_received_messages_in_last_hour() throws Exception {
```

Comment:
can we update this to use Should_ExpectedBehavior_When_StateUnderTest naming convention?